mailbbgun/views.py

```diff
@@ -0,0 +1,189 @@
+import flask
+import jsonschema
+import logging
+import pika
+
+from mailbbgun import app
+from mailbbgun import models
+
+
+if app.config['DEBUG']:
+    logging.basicConfig(level=logging.DEBUG)
+
+
+_DELIVERY_MODE_PERSISTENT = 2
+_LOG = logging.getLogger(__name__)
+
+
+class BadRequestException(Exception):
+    def __init__(self, message):
+        super().__init__(message)
+        self.message = message
+
+
+@app.route('/messages', methods=['POST'])
+def new_message():
+    data = _validate_message_request()
+    _LOG.debug("New message: {}".format(data))
+
+    message = models.Message(**data)
+    models.db.session.add(message)
+
+    connection = pika.BlockingConnection(
```

Comment:
No, unfortunately this is a very naive implementation that does create a new connection with each request.  I understand that this is inefficient, and adds a ton of overhead to each request.  What we need here is better integration with the flask framework, just like flask-sqlalchemy provides for sqlalchemy sessions.  I'll look around to see if there's any libs available that may work for this project and get back to you.